cmd/api/main.go

```diff
@@ -125,6 +136,9 @@ func main() {
 
 	// Shared options for the logger, with a custom gRPC code to log level function.
 	zapOpts := []grpc_zap.Option{
+		grpc_zap.WithDecider(func(fullMethodName string, err error) bool {
+			return fullMethodName != healthpb.Health_Check_FullMethodName
```

Comment:
This is added to prevent logging the health check requests, as these request will flood the logs and make it difficult for debugging. 
Is it necessary to log these requests?